src/pages/Landing/components/StudentTicket/Report/Form.js

```diff
@@ -0,0 +1,100 @@
+import React, { useReducer, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import attachment from '../../../../../assets/images/attachment.png'
+import styles from './Report.module.scss'
+
+const Form = () => {
+  //   const {user,setuser}=useState({
+  //     name:"",
+  //     title:"",
+  //     discription:"",
+  //   });
+  //   const {record,setrecord}=useState([]);
+  //   const handleInput=(e)=>{
+  //     const name=e.target.name;
+  //     const value=e.target.value;
+  //     console.log(name,value);
+  //     setuser({...user,[nsme]:value});
+  //   }
+  //   const Onsubmit=(e)=>{
+  //     e.preventDefault();
+  // const newRecord={...user};
+  // setrecord([...record,newRecord]);
+  //   }
+  return (
+    <form className={styles.Form}>
+      <div className={styles.Row}>
+        <h5 className={styles.Label}>Full Name</h5>
+        <input
+          name="Fullname"
+          type="text"
+          // value={user.name}
+          // onChange={handleInput}
+          placeholder=""
+        />
+      </div>
+      <div className={styles.Row}>
+        <h5 className={styles.Label}>Issue Title</h5>
+        <input
+          name="Issue Title"
+          type="text"
+          // value={user.title}
+          // onChange={handleInput}
+          placeholder=""
+        />
+      </div>
+      <div className={styles.Row}>
+        <h5 className={styles.Label}>Priority</h5>
+        <div className={styles.Buttons}>
+          <button type="button" className={styles.LowButtons}>
+            Low
+          </button>
+          <button type="button" className={styles.MediumButtons}>
+            Medium
+          </button>
+          <button type="button" className={styles.HighButtons}>
+            High
+          </button>
+        </div>
+      </div>
+      <div className={styles.Row}>
+        <h5 className={styles.Label}>Discription</h5>
+        <div className={styles.Wrapper}>
+          <textarea
+            name="description"
+            // value={user.description}
+            // onChange={handleInput}
+            type="text"
+            placeholder=""
+            className={styles.Discription}
+          />
+          <img src={attachment} alt="attach" />
```

Comment:
![image](https://user-images.githubusercontent.com/43199095/112511128-aff35b80-8dc4-11eb-9781-66eb90376071.png)
attach icon will be broken out when change size of textarea